metals-docs/src/main/scala/docs/UserConfigurationModifier.scala

```diff
@@ -62,7 +62,7 @@ class UserConfigurationModifier extends StringModifier {
                |**Example**:
                |```json
                |{
-               |  "metals.${option.key}": "${option.example}"
+               |  "metals.${option.camelCaseKey}": "${option.example}"
```

Comment:
You can't see it without expanding, but this is in the `lsp-config-coc` case, so it will only be used there.